### ISSUE TITLE
len_globals() returns memories.len()

### ISSUE THREAD

**Issue by graydon** [paritytech/wasmi#735]
I believe the code for `len_globals` is wrong here -- a copy-paste error from `len_memories` above:

https://github.com/paritytech/wasmi/blob/799995da7afb0ae02bb4fd1c9f242fe07c7ad824/crates/wasmi/src/module/mod.rs#L187-L189

**Comment by Robbepop** [comment]
@graydon good catch! very strange that this one was not caught before. Going to need further inspection.

**Comment by deepto98** [comment]
Can I work on this?

**Comment by Robbepop** [comment]
@deepto98 Yes please. :)
